src/transformers/models/blenderbot/tokenization_blenderbot.py

```diff
@@ -74,6 +74,23 @@ def build_inputs_with_special_tokens(self, token_ids_0: List[int], token_ids_1:
         """
         return token_ids_0 + [self.eos_token_id]
 
+    def _build_conversation_input_ids(self, conversation) -> List[int]:
```

Comment:
Could the type mention what is a conversation? I don't get it from that

Reply:
It's a `Conversation` which is defined way up in `pipelines/conversational.py`. Is there a way  to add it, without `importing`  ? (To avoid `import` loops )

Reply:
Actually it's the `Tensor` typing that I could add in some other places.

Reply:
You can do that by putting it into quotes: `"Conversation"`. IDEs should understand that.

Reply:
Ok, actually you still need to import it, but you can hide your import behind `TYPE_CHECKING`.
https://stackoverflow.com/questions/39740632/python-type-hinting-without-cyclic-imports

```python
if TYPE_CHECKING:
    import ...


def myfn(.... somevar: "SomeType")
```